providers/flagd/pkg/configuration.go

```diff
@@ -12,8 +12,11 @@ type ResolverType string
 
 // Naming and defaults must comply with flagd environment variables
 const (
-	defaultMaxCacheSize          int  = 1000
-	defaultPort                       = 8013
+	defaultMaxCacheSize  int = 1000
+	defaultRpcPort           = 8013
+	defaultInProcessPort     = 8015
+	// Deprecated: defaultPort is deprecated, please use either `defaultRpcPort` or `defaultInProcessPort`
+	defaultPort                       = defaultRpcPort
```

Comment:
IMO, no need to deprecate since `defaultPort` is internal. We can remove this safely.

Reply:
thank you, i removed it in https://github.com/open-feature/go-sdk-contrib/pull/524/commits/477a564b25b0312ac5c6d34c1d1f03321198a265